main/reference/zoe-api/user-seat.md

```diff
@@ -0,0 +1,105 @@
+# UserSeat Object
+
+Zoe uses **seats** to access or manipulate offers. They let contracts and users interact
+with them. Zoe has two kinds of seats. **ZCFSeats**
+are used within contracts and with **zcf** methods. **UserSeats** represent offers external to
+Zoe and the contract. The party who exercises an invitation and sends the **offer()** message
+to Zoe gets a **UserSeat** that can check payouts' status or retrieve the result of
```

Comment:
In one sense, I'm OK with concepts not being introduced in order in the API reference sections; these aren't meant to be read sequentially, from top to bottom. However, I thought it would help clarity in general to add "Payouts" to the "Zoe Data Types" topic, and have all instances of that word be a link to the data types topic.